app/controllers/extensions_controller.rb

```diff
@@ -15,10 +15,8 @@ class ExtensionsController < ApplicationController
   def index
     @extensions = @assessment.extensions.includes(:course_user_datum)
     @users = {}
-    @usersEncoded = {}
     @course.course_user_data.each do |cud|
       @users[cud.full_name_with_email] = cud.id
-      @usersEncoded[Base64.encode64(cud.full_name_with_email.strip).strip] = cud.id
```

Comment:
It seems like we have this Base64 encoding here for a reason, perhaps for students who username is not stored in base64 format? I will have to take a further look before deciding what's the appropriate fix here.

Reply:
Encoding is necessary to avoid XSS attacks